packages/teams-js/src/public/runtime.ts

```diff
@@ -89,6 +91,7 @@ export const teamsRuntimeConfig: IRuntime = {
   supports: {
     appInstallDialog: {},
     appEntity: {},
+    barCode: {},
```

Comment:
versionConstants below is where we would need to add barCode support if we want to maintain backcompat